src/components/PrList.js

```diff
@@ -82,12 +86,12 @@ class PrList extends React.Component {
               </thead>
               <tbody>
                 {pr_data_present
-                  ? (this.state.pr_data.map(function(object, i){
+                  ? (this.state.pageOfItems.map(function(object, i){
```

Comment:
@shekhar-amura : From where does the value of `pageOfItems` gets set when the page is loaded initially?

Reply:
we are updating pageOfItems from pagination component.  Once the pr_data is available then pagination component will pageOItems

Reply:
@shekhar-amura : Okay! We need to make sure that this `pagination` component is re-usable for any other component too before a single line of code change.